### ISSUE TITLE
Ancient dates in Gantt charts (or other timelines)

### ISSUE THREAD

**Issue by piperoc** [mermaid-js/mermaid#5695]
### Description

I would like to create a Gantt of events that occurred in the ancient past, where the date would fall in the Before Common Era (BCE).
I tried various ways but could not find any information or a way to do this.

The markdown would look something like:

```
gantt
    title Roman Kings Timeline
    dateFormat  YYYY
    axisFormat %Y

    Romulus: rm, 753 BCE, 37y
    Numa Pompilius: np, 715 BCE, 42y
    Tullius Hostilius: th, 673 BCE, 32y
```
Of course, this does not work since I do not know whether there is a syntax for such use case.
Maybe it is not possible.

### Steps to reproduce

1. type the following in the Live editor:
```
gantt
    title Roman Kings Timeline
    dateFormat  YYYY
    axisFormat %Y

    Romulus: rm, 753 BCE, 37y
    Numa Pompilius: np, 715 BCE, 42y
    Tullius Hostilius: th, 673 BCE, 32y
```

2. No diagram is displayed, but also _no error is reported by the editor_.

### Setup

- Mermaid version:
- Browser and Version: [Chrome, Edge, Firefox]

**Comment by piperoc** [comment]
I have the feeling that the limitation might be to the Unix Timestamp which only counts from 1/1/1970...

**Comment by piperoc** [comment]
Someone answered me on [StackOverflow ](https://stackoverflow.com/questions/78839768/is-there-a-way-to-represent-a-ancient-date-bce-in-mermaid-timeline-or-gantt-ch) so I used the JavaScript notation for "negative" dates. Here's what I got.

```
gantt
    title Roman Kings Timeline
    dateFormat  YYYY
    axisFormat %Y BCE

    Romulus: rm, -00753, 37y
    Numa Pompilius: np, -00715, 42y
    Tullius Hostilius: th, -00673, 32y
```

Which is not perfect but at least it's more accurate.

I don't know if there is **_an idea for the year out format_**, which is not a showstopper anyway.
